etc/lfcd.ps1

```diff
@@ -13,14 +13,4 @@
 # You may put this in one of the profiles found in $PROFILE.
 #
 
-$tmp = [System.IO.Path]::GetTempFileName()
-lf -last-dir-path="$tmp" $args
-if (Test-Path -PathType Leaf "$tmp") {
-    $dir = Get-Content "$tmp"
-    Remove-Item -Force "$tmp"
-    if (Test-Path -PathType Container "$dir") {
-        if ("$dir" -ne "$pwd") {
-            cd "$dir"
-        }
-    }
-}
+lf -print-last-dir $args | cd
```

Comment:
Very minor, but I suggest to avoid using aliases in PowerShell scripts:

```suggestion
lf -print-last-dir $args | Set-Location
```

Apart from that, looks good.

Reply:
Totally agree. `cd` was used before, so i left it as it is.